### PR TITLE
Move all the files under /public

### LINKED ISSUE #7: Refactor temp/assets/...
Move the contents in this directory into /assets

e.g. /temp/assets/css/ --> /assets/css

### LINKED ISSUE #9: Refactor review page
reviewPage.css, reviewPage.html

### LINKED ISSUE #10: Refactor PR Waiting Room
prWaitingRoom.css
prWaitingRoom.html

### LINKED ISSUE #11: Refactor patWaitingRoom
patWaitingRoom.html
patWaitingRoom.css

### PR DESCRIPTION
**Pull request by lillianchou94**
There's no easy way to depend a PR on another PR... please review by commit :)

### CODE REVIEW

File: scripts/patientSignUp.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ function formSubmit() {
 }
 
 if(formSubmit()) {
-    window.location = "/FirebaseRTC/public/patWaitingRoom.html";      
+    window.location = "/templatespublic/patWaitingRoom.html";
```
Comment (j0n3lson) [inline scripts/patientSignUp.js]: Missing `/` here, should be:

`/templates/public`

Ditto line 81.

File: scripts/patientSignUp.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ if(formSubmit()) {
 //     }
 //     if (i > 1) {  
 //         alert('Is Working');
-//         window.location = "https://hackathonv4.glitch.me/FirebaseRTC/public/patWaitingRoom.html";      
+//         window.location = "https://hackathonv4.glitch.me/public/templates/patWaitingRoom.html";
```
Comment (j0n3lson) [inline scripts/patientSignUp.js]: This is a comment but the url is wrong here. I think the reference needs to remain since the projecton glitch hasn't changed. 

Either delete this comment or don't change the URL here.

File: public/templates/finesse_match.html [hunk 3]
```diff
@@ -47,7 +47,7 @@
 
     <!-- import the webpage's javascript file -->
     <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-    <script src="/scripts/patientSignUp.js" defer></script>
+    <script src="/public/components/patientSignUp.js" defer></script>
```
Comment (j0n3lson) [inline public/templates/finesse_match.html]: Should this go into `/public/assets/js/` it's ordinary JS as opposed to js that interacts with a system component like firebase services.

I think my original intent behind `/public/components/` was to store stuff that is either a self-contained subsystem or code that interacts with dependencies in some way.

File: public/templates/index.html [hunk 4]
```diff
@@ -124,21 +124,21 @@ <h5>For Practioners:</h5>
                     <p>If you are a practitioner sign up to offer free assistance to those in need. All credentials will be verified. </p>
                     <ul>
                         <li>
-                            <img src="/assets/images/about-icon-01.png" alt="">
+                            <img src="/public/assets/images/images/about-icon-01.png" alt="">
```
Comment (j0n3lson) [inline public/templates/index.html]: Looks like the `src` path is wrong here. 

You actually moved the image files to `/public/asset/images` but your source has it going to `/public/asset/images/images`.

Ditto all the relative paths in this commit.

File: public/components/patientSignUp.js [hunk 5]
```diff
@@ -101,7 +101,7 @@ if(formSubmit()) {
 //     }
 //     if (i > 1) {  
 //         alert('Is Working');
-//         window.location = "https://hackathonv4.glitch.me/FirebaseRTC/public/patWaitingRoom.html";      
+//         window.location = "https://hackathonv4.glitch.me/public/templates/patWaitingRoom.html";
```
Comment (j0n3lson) [inline public/components/patientSignUp.js]: If this comment is going to stay, I think this line needs to be the same. The glitch project hasn't actually changed.

Either delete the comment or revert the change on this line so that at least the comment is logically correct.

File: public/templates/index.html [hunk 6]
```diff
@@ -124,21 +124,21 @@ <h5>For Practioners:</h5>
...
                             <div class="text">
                                 <h6>Get Verified</h6>
                                 <p>To sign up as a medical practioner, you must upload photocopies of your medical license, degree and other important credentials. Our team at Melly will then verify your credentials.</p>
                             </div>
                         </li>
                         <li>
-                            <img src="/assets/images/about-icon-02.png" alt="">
+                            <img src="/public/assets/images/images/about-icon-02.png" alt="">
```
Comment (j0n3lson) [inline public/templates/index.html]: This is incorrect. The files were moved to "/public/assets/images" and there is no `images` sub-folder under that folder.

So it should be `/public/assets/images/...` and ditto elsewhere.

**Review by j0n3lson — CHANGES_REQUESTED** [review]

File: public/templates/finesse_match.html (hunk 3 above)
Comment (lillianchou94) [reply public/templates/finesse_match.html]: Could we keep them here for the time being as we work through the JS code for firebase? I suspect we might need a bit more discussion around this

File: public/templates/index.html (hunk 4 above)
Comment (lillianchou94) [reply public/templates/index.html]: Done

File: public/components/patientSignUp.js (hunk 5 above)
Comment (lillianchou94) [reply public/components/patientSignUp.js]: Done

File: public/templates/finesse_match.html (hunk 3 above)
Comment (j0n3lson) [reply public/templates/finesse_match.html]: I think that's fine.

**Review by j0n3lson — DISMISSED** [review]
I think you called out an issue today where using `/public` in the href doesn't work when we set `public` as the firebase root.

I guess these need to be updated to remove `/public/` from hrefs?

**Comment by j0n3lson** [comment]
@lillianchou94 I think you want to rebase in order to get the new per-pr preview goodness :smile: :rocket:

**Review by j0n3lson — APPROVED** [review]
